consensus/polybft/state.go

```diff
@@ -79,8 +81,11 @@ func (s *State) initStorages() error {
 		if err := s.ProposerSnapshotStore.initialize(tx); err != nil {
 			return err
 		}
+		if err := s.StakeStore.initialize(tx); err != nil {
```

Comment:
How was this initialized previously?

Reply:
I am not sure what you mean. Previous code was:
`return s.StakeStore.initialize(tx)`

Now it is:

```
if err := s.StakeStore.initialize(tx); err != nil {
     return err
}

return s.GovernanceStore.initialize(tx)
```